src/components/ClassicArrow/Arrow.tsx

```diff
@@ -87,29 +87,42 @@ export const ClassicArrow: React.FC<ClassicArrowProps> = ({
   }, [editItem, item.id, updateArrowType]);
 
   const isHorizontal =
-    Math.abs(item.startPosition.x - item.endPosition.x) >
-    Math.abs(item.startPosition.y - item.endPosition.y);
+    item.startPosition != null && item.endPosition != null
+      ? Math.abs(item.startPosition.x - item.endPosition.x) >
+        Math.abs(item.startPosition.y - item.endPosition.y)
+      : true;
 
   const transform = isHorizontal
     ? `translateY(-${gapSize / 2}px)`
     : `translateX(-${gapSize / 2}px)`;
 
-  const xAdjustStart = xAdjustmentStart(item, isHorizontal);
-  const yAdjustStart = yAdjustmentStart(item, isHorizontal);
-
-  const startPos = {
-    x: (item.startGridPosition.x + xAdjustStart) * (cellSize + gapSize),
-    y: (item.startGridPosition.y + yAdjustStart) * (cellSize + gapSize),
+  // Make old arrows visible in grid, so that they can be deleted
```

Comment:
```suggestion
  // Make old arrows visible in grid, so that they can be deleted
  // TODO: Create a new major/minor versjon, delete these arrows in `upgrades.json` and remove this code
```